### DISCUSSION TITLE
Grove Inventor Kit for microbit | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#162] [Q&A]
# Grove Inventor Kit for microbit | Seeed Studio Wiki

Grove_Inventor_Kit_for_microbit

https://wiki.seeedstudio.com/Grove_Inventor_Kit_for_microbit/

**Comment by matsujirushi** [comment]
I cannot download "Grove Shield for microbit_eagle file.zip".

<img width="857" height="131" alt="image" src="https://github.com/user-attachments/assets/c18cc059-34e4-416d-a1bb-abb5fe69be37" />
